test/rubygems/test_gem_ext_cargo_builder_unit.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+require_relative 'test_gem_ext_cargo_builder'
+require 'rubygems/ext'
+
+class TestGemExtCargoBuilderUnit < Gem::TestCase
+  def test_cargo_command_passes_args
+    skip_unsupported_platforms!
```

Comment:
Isn't it possible to move this to `setup` since it is shared for all tests? Does skipping of tests work in there?

Reply:
done!

Reply:
tried this, and unfortunately it did not work :/

Reply:
I wasn't sure, sorry for confusion.